keyboards/tr60w/readme.md

```diff
@@ -0,0 +1,8 @@
+This is the hotswap pcb supplied by triangle labs with their tr60 case.
```

Comment:
Could you reformat this file to align to the [AVR readme template](https://github.com/qmk/qmk_firmware/blob/b663a5b2486eb47909956b77d6dd13e08400f2fe/quantum/template/avr/readme.md)? (It's my fault for not specifying that we have one. :blush:)

Also, `Triangle` is misspelled on the last line.